Meadowlark-js/backends/meadowlark-opensearch-backend/src/repository/OpenSearchException.ts

```diff
@@ -6,7 +6,7 @@ import { OpenSearchClientError, ResponseError } from '@opensearch-project/opense
 import { Logger } from '@edfi/meadowlark-utilities';
 import { QueryResult } from '@edfi/meadowlark-core';
 
-export async function LogOpenSearchErrors(
+export async function handleOpenSearchError(
```

Comment:
renamed because it doesn't only log. it also decides on a result to return in the case of an error.